### ISSUE TITLE
 @import *.css not work error: Can't resolve './style.css'

### ISSUE THREAD

**Issue by L1uD0ngq1** [webpack-contrib/sass-loader#629]
Files Tree Demo:
src/index.scss
src/styles/index.scss
src/styles/ztree.css

And relationship:
src/index.scss => @import("./styles/index.scss")
src/styles/index.scss => @import("./ztree.css")

webpack config:
```javascript
 {    loaders: [{
        loader: 'css-loader',
        options: {
          modules: !!config.CSSModules,
          importLoaders: 1,
          // minimize: true, // There is not obvious change on file size after minimize.
          localIdentName: '[name]__[local]___[hash:base64:5]',
          root: config.publicPath
        },
      }, {
        loader: 'postcss-loader',
        options: {
          config: {
            path: paths.ownPostCSSConfig,
          }
        }
      }, {
        loader: 'sass-loader',
        query: {
          sourceMap: true,
          includePaths : [ 'src' ]
        },
      }]
}
```

It works well if ztree is a sass file, however it throw error when ztree is a css file;
![image](https://user-images.githubusercontent.com/42128808/48536831-b1eeab00-e8ea-11e8-9e27-a5ed40315c11.png)

I try to research it, and find sass-loader does not include it just link it. then it will lose the real path. 
![image](https://user-images.githubusercontent.com/42128808/48536914-e3677680-e8ea-11e8-909b-d387e98e3699.png)

Another thing confuse me is that webpackImporter's commet tell me: 
![image](https://user-images.githubusercontent.com/42128808/48537060-507b0c00-e8eb-11e8-9f66-ab4b0ad8b1d3.png)

Could you explain this two comments? And how to resolve my issue?

**Comment by L1uD0ngq1** [comment]
It's easy to reproduce it, i didn't write the demo. The images are from my product project.

**Comment by ryuran** [comment]
It should be fixed by node-sass 4.11
try `npm install node-sass@beta --save-dev`

**Comment by evilebottnawi** [comment]
@L1uD0ngq1 friendly ping
